client/json-rpc/src/client.rs

```diff
@@ -120,11 +120,20 @@ pub struct JsonRpcAsyncClient {
 }
 
 impl JsonRpcAsyncClient {
+    /// Deprecated, use `new_from_url`
     pub fn new(client: Client, host: &str, port: u16) -> Self {
```

Comment:
Silly question number 2 😄What's the reason behind deprecating this call? Also if you want to deprecate it, can we not just remove the method and update all the call sites?

Reply:
you also have this thing I believe: https://doc.rust-lang.org/edition-guide/rust-2018/the-compiler/an-attribute-for-deprecation.html

Reply:
done, deleted all call site that uses the old one